prime-router/src/main/kotlin/metadata/Mappers.kt

```diff
@@ -529,6 +529,48 @@ class LookupSenderValuesetsMapper : Mapper {
     }
 }
 
+/**
+ * The lookupSenderAutomationValuesetsMapper is used to lookup values from the
+ *      "sender_automation_value_set_row" table/csv
+ * The args for the mapper are:
+ *      args[0] --> senderAutomationValueSetId = the id of the sender automation value set from
+ *          sender_automation_value_set primary lookup field
+ * The mapper uses the above arguments + the question's answer to retrieve a row from the table
+ */
+class LookupSenderAutomationValuesetsMapper : Mapper {
+    override val name = "lookupSenderAutomationValuesetsMapper "
+
+    override fun valueNames(element: Element, args: List<String>): List<String> {
+        return args
+    }
+
+    override fun apply(
+        element: Element,
+        args: List<String>,
+        values: List<ElementAndValue>,
+        sender: Sender?
+    ): ElementResult {
+        return ElementResult(
+            if (values.size != args.size) {
+                null
+            } else {
+                val lookupTable = element.tableRef
+                    ?: error("Schema Error: could not find table ${element.table}")
+                val tableFilter = lookupTable.FilterBuilder()
+                val senderAutomationValueSetId = args[0]
```

Comment:
I think you mean this to be the numeric ID.  Instead of a numeric ID use the valueset name to make it easier to read a schema.